docs/en/latest/tutorials/expose-api.md

```diff
@@ -111,7 +111,7 @@ curl "http://127.0.0.1:9180/apisix/admin/routes/1" \
 After creating the Route, you can test the Service with the following command:
 
 ```
-curl -i -X GET "http://127.0.0.1:9080/get?foo1=bar1&foo2=bar2" -H "Host: example.com"
+curl -i -X GET "http://127.0.0.1:9080/anything/get?foo1=bar1&foo2=bar2" -H "Host: example.com"
 ```
 
 APISIX will forward the request to `http://httpbin.org:80/anything/foo?arg=10`.
```

Comment:
Yes, the route uri is `/anything/*`, so you have to use `/anything` as prefix.
Besides your fix, you need to fix the url of this sentence to `http://httpbin.org:80/anything/get?foo1=bar1&foo2=bar`.

Reply:
Thanks! I'll update it.

Reply:
@kingluo please have a look now! 😄

Reply:
@Neilblaze You need to fix the uri in the route cfg in getting-started.md too:

```bash
curl "http://127.0.0.1:9180/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f136f87ad84b625c8f1" -X PUT -d '
{
  "uri": "/get",
  "host": "httpbin.org",
  "upstream_id": "1"
}'
```

Reply:
@navendu-pottekkat I discovered something interesting. In the master branch, `getting-started.md` only exists in _[Mandarin](https://github.com/apache/apisix/blob/master/docs/zh/latest/getting-started.md)_ (zh) & **not** in [English](https://apisix.apache.org/docs/apisix/getting-started/), but on doing some digging, I found that the English version of the same can be discovered in **[`release/3.2`](https://github.com/apache/apisix/blob/release/3.2/docs/en/latest/getting-started.md)** branch. Should I create a copy of the english version of `getting-started.md` in _**master**_ too?


Also, @kingluo are you highlighting **[this](https://github.com/apache/apisix/blob/12e9628a1ba9dac166e507ebe0c9eb47c82364dc/docs/en/latest/getting-started.md?plain=1#L229-L235)** part for the proposed changes? Thanks! :)

Reply:
Check the Chinese doc.